examples/Demo/FluentUI.Demo.Client/Documentation/Components/Spacer/Examples/SpacerWidth.razor

```diff
@@ -0,0 +1,14 @@
+﻿<div style="display: flex; width=100%">
+    <FluentIcon Value="@(new Icons.Filled.Size48.Alert())" Color="@Color.Neutral" />
+    <FluentSpacer Width="@Width" />
```

Comment:
This is an example, so that will be easier for the dev to see `<FluentSpacer Width="50px" />` (and not adding a variable)